docs/tutorials/notebooks/Unbalanced_OT.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def plot_samples(source, target):
</pre><p>you are never using this function again. I think that in an illustrative notebook it is better to not write this as a function then.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/291/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>